README.md

```diff
@@ -1,4 +1,9 @@
 # Plonk Gadgets
+![Build Status](https://github.com/dusk-network/plonk/workflows/Continuous%20integration/badge.svg)
+[![Repository](https://img.shields.io/badge/github-plonk-blueviolet?logo=github)](https://github.com/dusk-network/plonk)
+[![Documentation](https://img.shields.io/badge/docs-plonk-blue?logo=rust)](https://docs.rs/plonk/)
```

Comment:
They points at plonk repo, not plonk_gadgets. Looks a bit weird to me, is that a copy & paste typo or is correct?

Reply:
Yes... ahhahahaha that's the problem of rushing. Nice catch!!!

Will amend it now!